XDSP/XDSP.h

```diff
@@ -93,34 +94,34 @@ namespace XDSP
         using namespace DirectX;
 
         // sign constants for radix-4 butterflies
-        static const XMVECTORF32 vDFT4SignBits1 = { { { 1.0f, -1.0f,  1.0f, -1.0f } } };
```

Comment:
While not required for Visual C++, the extra braces here are actually needed for C++ conformance.

With clang/LLVM, not having these extra braces results in ``-Wmissing-braces``.

Please put them back for the various ``XMVECTORF32`` cases you removed them from.

Reply:
Thank you for reviewing.
I put extra braces back for clang/LLVM and re-committed. At that time, the indent contained tabs, so I committed twice to fix it.

I also tested a combination of each platform/configuration and compiler switch using the directxmathtest/XDSPTest you pointed to.
And I got the following results without warnings in all of them:

FFT4 passed
FFT8 passed
FFT16 passed
FFT (32) passed
FFTUnswizzle passed
FFTPolar passed
Deinterleave passed
Interleave passed
FFTInterleaved passed
IFFTDeinterleaved passed